<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmulatorMainWidget</class>
 <widget class="QWidget" name="EmulatorMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FreeHeroes - Battle Emulator</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout" stretch="0,1,0,0,0">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>50</number>
       </property>
       <property name="verticalSpacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Terrain</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="FreeHeroes::Gui::ResizeableComboBox" name="comboBoxTerrain"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Obstacles preset</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="FreeHeroes::Gui::ResizeableComboBox" name="comboBoxObstaclePreset"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="horizontalSpacing">
        <number>50</number>
       </property>
       <property name="verticalSpacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Battle positions preset</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="FreeHeroes::Gui::ResizeableComboBox" name="comboBoxPositionsPreset"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Defender army preset</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="FreeHeroes::Gui::TreeComboBox" name="comboBoxObjectPreset"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="FreeHeroes::BattleEmulator::ArmyConfigWidget" name="armyConfigAtt" native="true"/>
     </item>
     <item>
      <widget class="FreeHeroes::BattleEmulator::ArmyConfigWidget" name="armyConfigDef" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Replays:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FreeHeroes::Gui::ResizeableComboBox" name="comboBoxReplaySelect"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRenameReplay">
       <property name="text">
        <string>Rename</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClearReplays">
       <property name="text">
        <string>Clear all replays</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReplayLoadAdv">
       <property name="text">
        <string>Load Adventure data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>25</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxEnableRecording">
       <property name="text">
        <string>Enable replay recording</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonReplayBattle">
       <property name="text">
        <string>Replay battle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButtonSettings">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRegenMana">
       <property name="text">
        <string>Regenerate mana
for all heroes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonNextDay">
       <property name="text">
        <string>Next day
adventure mode</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxQuickCombat">
       <property name="text">
        <string>Quick combat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="FreeHeroes::Gui::AdventureKingdomStatusWidget" name="kingdomStatusWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FreeHeroes::Gui::ResizeableComboBox</class>
   <extends>QComboBox</extends>
   <header>CustomFrames.hpp</header>
  </customwidget>
  <customwidget>
   <class>FreeHeroes::Gui::TreeComboBox</class>
   <extends>QComboBox</extends>
   <header>CustomFrames.hpp</header>
  </customwidget>
  <customwidget>
   <class>FreeHeroes::BattleEmulator::ArmyConfigWidget</class>
   <extends>QWidget</extends>
   <header>ArmyConfigWidget.hpp</header>
  </customwidget>
  <customwidget>
   <class>FreeHeroes::Gui::AdventureKingdomStatusWidget</class>
   <extends>QWidget</extends>
   <header>AdventureKingdomStatusWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
